core/network/fabric-interop-cc/libs/assetexchange/README.md

```diff
@@ -3,4 +3,194 @@
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# Asset Exchange Library
\ No newline at end of file
+# Asset Exchange Library
+
+To use this library, in you smart contract `go.mod`, add following in require (update the version according to the latest module version):
+```
+require(
+    ...
+    github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go v1.5.3
+    github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/assetexchange v1.5.3
+    ...
+)
+```
+
+There are two kinds of APIs of library, one with `contractId`, and other without `contractId`.
```

Comment:
Replace with: `The asset exchange library within the Weaver chaincode package offers two sets of APIs to manage asset locks and claims for a cross-ledger asset exchange. In one set, the transactions take a unique contract ID referring to an asset exchange instance as parameter. This contract ID is generated by the Weaver chaincode when the first asset is locked (as per the protocol steps), and returned to the calling application client. Though you must use contract ID-based API functions for fungible assets, you may also use them for non-fungible assets. For non-fungible assets, Weaver also provides a parallel set of API functions that take asset type and ID as parameters rather than a temporarily generated contract ID.`